Tuple is returned as list when interpolating

```
from config import config

def test_tuple_interpolate():
    DICT = {"a1": (1, 2)}
    cfg = config(DICT, interpolate=True)
    assert cfg["a1"] == (1, 2)
```
Results in:
```
>       assert cfg["a1"] == (1, 2)
E       assert [1, 2] == (1, 2)
E         Full diff:
E         - (1, 2)
E         + [1, 2]
```

--- a/config/helpers.py
+++ b/config/helpers.py
@@ -198,6 +198,9 @@ def interpolate_object(
         else:
             raise ValueError('Invalid interpolation method "%s"' % method)
     elif hasattr(obj, "__iter__"):
-        return [interpolate_object(attr, x, d, method) for x in obj]
+        if isinstance(obj, tuple):
+            return tuple(interpolate_object(attr, x, d, method) for x in obj)
+        else:
+            return [interpolate_object(attr, x, d, method) for x in obj]
     else:
         return obj

--- a/tests/test_datatypes.py
+++ b/tests/test_datatypes.py
@@ -55,3 +55,9 @@ def test_attribute_dict_1():  # type: ignore
 
     d.var3 = "abc"
     assert d.var3 == "abc"
+
+
+def test_tuple():
+    DICT = {"a1": (1, 2)}
+    cfg = config_from_dict(DICT, interpolate=True)
+    assert cfg["a1"] == (1, 2)

